src/Bridges/ApplicationDI/ApplicationExtension.php

```diff
@@ -56,10 +64,16 @@ public function loadConfiguration()
 			$application->addSetup('Nette\Bridges\ApplicationTracy\RoutingPanel::initializePanel');
 		}
 
-		$touchToRebuild = $this->debugMode && $config['scanDirs'] ? reset($config['scanDirs']) : NULL;
+		$code = '!($me = $this) ?: ' . $container->formatPhp(
+			'function ($class) use ($me) { return Nette\Bridges\ApplicationDI\Helpers::createPresenter($me, $class, ?, ?); }',
```

Comment:
Yes, it is fine. I take this more as a cosmetic detail, which can be changed at any time.